frontend/src/reducers/user.js

```diff
@@ -0,0 +1,33 @@
+import { createSlice } from "@reduxjs/toolkit"
+
+const initialState = {
+    login: {
+      accessToken: null,
+      userId: 0,
+      statusMessage: "",
+    },
+  }
+  
+  export const user = createSlice({
+    name: "user",
+    initialState: initialState,
+    reducers: {
+      setAccessToken: (state, action) => {
+        const { accessToken } = action.payload
+        state.login.accessToken = accessToken
+      },
+      setUserId: (state, action) => {
+        const { userId } = action.payload
+        state.login.userId = userId
+      },
+      setStatusMessage: (state, action) => {
+        const { statusMessage } = action.payload
+        state.login.statusMessage = statusMessage
+      },
+      setLogout: (state, action) => {
+        state.login.userId = 0
+        state.login.accessToken = null
+        state.login.statusMessage = "logged out"
+      },
+    },
+  })
```

Comment:
Reducer is a great place to manage your users but you need to save some data in the local storage to keep your users logged in after they refresh the page.